apps/meteor/app/livechat/imports/server/rest/users.js

```diff
@@ -20,41 +29,42 @@ API.v1.addRoute(
 			const { text } = this.queryParams;
 
 			if (this.urlParams.type === 'agent') {
+				if (!(await hasAtLeastOnePermission(this.userId, ['transfer-livechat-guest', 'edit-omnichannel-contact']))) {
```

Comment:
either `await` for the `async` version or don't use `await`
```suggestion
				if (!(await hasAtLeastOnePermissionAsync(this.userId, ['transfer-livechat-guest', 'edit-omnichannel-contact']))) {
```